other/upgrade.php

```diff
@@ -5757,4 +5757,4 @@ function upgradeGetColumnInfo($targetTable, $column): array
 	return [];
 }
 
-?>
\ No newline at end of file
+https://youtu.be/u9Dg-g7t2l4?>
```

Comment:
🤨

Reply:
That's what happens when I accidentally middle click on my laptop instead of left click. Not sure how I didn't notice but I'll get rid of it